### PR TITLE
Add tutorial to train CXN with attention

### PR DESCRIPTION
**Pull request by ninamiolane**

### CODE REVIEW

File: docs/notebooks/convcxn_train.ipynb [hunk 1]
```diff
@@ -7,7 +7,7 @@
    "source": [
     "# Tutorial: Set-up, create and train a convolutional CXN\n",
     "\n",
-    "In this notebook, we create and train a simplified, non-attentional version of a CXN network, originally proposed in the paper by Hajij et. al : Cell Complex Neural Networks (https://arxiv.org/pdf/2010.00743.pdf). We will load a cellular complex dataset from the web and train the model to perform classificaiton on this dataset."
+    "In this notebook, we create and train a simplified, non-attentional version of a CXN network, originally proposed in the paper by Hajij et. al : Cell Complex Neural Networks (https://arxiv.org/pdf/2010.00743.pdf). We will load a cell complex dataset from the web and train the model to perform classificaiton on this dataset."
```
Comment (mathildepapillon) [inline docs/notebooks/convcxn_train.ipynb:10]: "non-attentional version of a CXN Network" --> "attentional CXN Network"

File: tutorials/cxn_train.ipynb [hunk 2]
```diff
@@ -7,12 +7,12 @@
    "source": [
     "# Tutorial: Set-up, create and train a convolutional CXN\n",
     "\n",
-    "In this notebook, we create and train a simplified, non-attentional version of a CXN network, originally proposed in the paper by Hajij et. al : Cell Complex Neural Networks (https://arxiv.org/pdf/2010.00743.pdf). We will load a cellular complex dataset from the web and train the model to perform classificaiton on this dataset."
+    "In this notebook, we create and train a simplified, non-attentional version of a CXN network, originally proposed in the paper by Hajij et. al : Cell Complex Neural Networks (https://arxiv.org/pdf/2010.00743.pdf). We will load a cell complex dataset from the web and train the model to perform classificaiton on this dataset."
```
Comment (mathildepapillon) [inline tutorials/cxn_train.ipynb:10]: "non-attentional version of a CXN Network" --> "attentional CXN Network"

File: docs/notebooks/convcxn_train.ipynb (hunk 1 above)
Comment (ninamiolane) [reply docs/notebooks/convcxn_train.ipynb]: Done. Also the notebooks/ folder shouldn't be a folder but a symlink. I've fixed that in another PR.

File: tutorials/cxn_train.ipynb (hunk 2 above)
Comment (ninamiolane) [reply tutorials/cxn_train.ipynb]: Done.

**Pull request merged**